Source/Tests/LinuxSyscalls/FileManagement.h

```diff
@@ -78,6 +80,7 @@ class FileManager final {
   void UpdatePID(uint32_t PID) { CurrentPID = PID; }
 
   std::string GetEmulatedPath(const char *pathname, bool FollowSymlink = false);
+  std::optional<std::pair<int, const char*>> GetEmulatedFDPath(const char *pathname, bool FollowSymlink, std::array<char[PATH_MAX], 2> &TmpFilename);
```

Comment:
perhaps the

```cpp
std::array<char[PATH_MAX], 2>
```

should be extracted into a `using` alias, so it doesn't need to written out long-form in several cases and keeps the underlying type definition in one place

Looks good otherwise

Reply:
Switched over to a `using` alias